src/components/ApplicationForm/ReviewCards.js

```diff
@@ -188,7 +188,14 @@ export default ({ formInputs, handleEdit, onChange }) => {
         </JohnDiv>
         <StyledBanner wide={true} blur>
           <ContentWrapper>
-            <InfoGroup heading="Resume" data={formInputs.skills.resume.match(/[/\\]([\w\d\s.\-()]+)$/)[1]} />
+            <InfoGroup
+              heading="Resume"
+              data={
+                (formInputs.skills.resume &&
+                  formInputs.skills.resume.match(/[/\\]([\w\d\s.\-()]+)$/)[1]) ||
+                ''
```

Comment:
Does passing in `true && <resume_name>` just pan out to `<resume_name`>? Haven't seen this syntax before but if it does then all good! My suggestion would be:
```
data={
  formInputs.skills.resume ?
  formInputs.skills.resume.match(/[/\\]([\w\d\s.\-()]+)$/)[1]) :
  ''
}
```

Reply:
Yup! Good suggestion - I'll change it :)